app/mailers/patient_mailer.rb

```diff
@@ -189,7 +189,7 @@ def closed_email(patient)
 
     @patient = patient
     @lang = patient.select_language
-    mail(to: patient.email&.strip, subject: I18n.t('assessments.email.closed.subject', locale: @lang || :en)) do |format|
+    mail(to: patient.email&.strip, subject: I18n.t('assessments.email.closed.subject', locale: @lang || :eng)) do |format|
```

Comment:
`|| :eng` is not required because `.select_language` is going to return `:eng` as default.

Reply:
good catch! missed this one. Removed.